src/views/AddExpenses.js

```diff
@@ -20,66 +26,104 @@ const AddExpenses = ({ addExpense, setMonthToShow }) => {
     setTitle('');
     setDate('');
     setAmount('');
+    setWasTitleValidated(false);
+    setWasDateValidated(false);
+    setWasAmountValidated(false);
+  };
+
+  const handleTitleChange = evt => {
+    setTitle(evt.target.value);
+    setWasTitleValidated(true);
+  };
+
+  const handleDateChange = date => {
+    setDate(date);
+    setWasDateValidated(true);
+    setMonthToShow(
```

Comment:
this is somewhat unrelated to this merge request but it got me really confused. here we call the callback to `setMonthToShow`. The name suggests that we instantly change the month that will be shown on the page (incl. the expense history list). this would not make sense to me. however, the actual thing determining which month is currently showing is `App.js#selectedMonth`. anyway, this got me really confused. maybe we can update the naming a little to clear up what is going on?

Reply:
renamed to `newExpenseMonth` and `setNewExpenseMonth` and added brief comment